texastribune/donations-app
# Mail-a-check info addition

#### What's this PR do?

Adds option to grab membership form and use it to mail in a check, if someone prefers to become a member or donate that way.
#### Why are we doing this? How does it help us?

Helps us make sure people who want to mail in checks can find that option.
#### How should this be manually tested?

Run the app locally.

Visit the index page. Look at the bottom of the right sidebar for "Prefer to mail a check?" section. It should be below the "Want to give more?" question at the bottom. Check it out on mobile, too. Click where it says "click here" to see that the membership form opens as expected for you.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

Not really.
#### What are the relevant tickets?

Sprint request.
#### Next steps?
#### Smells?
#### TODOs:
- [x] Get art approval (sent screenshots to Ben)
#### Has the relevant documentation/wiki been updated?
#### Technical debt note

Slightly more
